simulation/traffic_simulator/src/hdmap_utils/hdmap_utils.cpp

```diff
@@ -866,6 +959,16 @@ std::vector<std::int64_t> HdMapUtils::getNextLaneletIds(
   return ret;
 }
 
+std::vector<std::int64_t> HdMapUtils::getNextLaneletIds(
+  const std::vector<std::int64_t> & lanelet_ids, const std::string & turn_direction) const
+{
+  std::vector<std::int64_t> ret;
+  for (const auto & id : lanelet_ids) {
+    ret = concat(ret, getNextLaneletIds(id, turn_direction));
+  }
+  return ret;
```

Comment:
It seems to be a bug that this function does not use `sortAndUnique`.

Reply:
Sorry, you are right.
It is bug.

Reply:
Sorry, you are right.
It is bug.

Reply:
[retrun sortAndUnique result.](https://github.com/tier4/scenario_simulator_v2/blob/a66dbee757d97e7f6e793b24e84dad9ac86227d4/simulation/traffic_simulator/src/hdmap_utils/hdmap_utils.cpp#L950)